src/Server.js

```diff
@@ -35,6 +37,11 @@ function middleware(bundler) {
       }
     }
 
+    function send500() {
+      res.writeHead(500);
```

Comment:
I'd love if this responded with a trimmed down bundle that included the hmr-runtime and reloaded when the build succeeded but I couldn't figure out how to make it work. Any ideas?